### DISCUSSION TITLE
问问大家用的都是什么设备啊(｡•̀ᴗ-)✧

### DISCUSSION THREAD

**Discussion by Jason555647** [share121/inter-knot#5119] [灌水]
![Uploading mmexport1726586533618.jpg…]()

**Comment by 952E-348A** [comment]
手机。

**Comment by jiezai23** [comment]
手机，vivo的。

**Comment by keli-star** [comment]
汽车车机

**Reply by Jason555647** [reply]
重量级（doge

**Comment by rtcrtc123** [comment]
手机，电脑

**Comment by Rjjfk** [comment]
平板

**Comment by Andrea0126** [comment]
手機和電腦

**Comment by cpu644** [comment]
电脑。这个东西在ios上懒得搞

**Reply by Yizakl** [reply]
🤔挺简单的

**Comment by Yizakl** [comment]
iOS

**Reply by EcoFur** [reply]
你好活跃！
你才是大佬

**Reply by Yizakl** [reply]
沃市FVV

> 你好活跃！
> 你才是大佬

**Comment by BaiHan2333** [comment]
手机

**Comment by qingmoz123** [comment]
手机和笔记本

**Comment by qingmoz123** [comment]
手机和笔记本

**Comment by Orangered81** [comment]
电子书

**Comment by LeoneMafia** [comment]
很简单，手机

**Comment by yuexiayin** [comment]
OPPOa5

**Comment by hoyotao** [comment]
Xiaomi Pad 6S Pro 12.6

**Reply by Jason555647** [reply]
嘿嘿，我也是xiaomi平板

**Comment by TNCNET** [comment]
移动端 红米k50
OS version 1.0.10.0/Hyperos
Soc=8100

PC Windows11 /24H2/build26120
7700HQ+1050/4GB

**Comment by RWHwaxili** [comment]
华擎主板，AMD的U，580卡皇，三星的内存

**Comment by Almost-Noob** [comment]
用帮布比用手机方便，只用在帮布上装一套投影模组，直接大屏刷绳网。

**Comment by fzdqz** [comment]
用的是手机哦

**Comment by QuasarRyan** [comment]
MacBook Pro 2021 14寸，环境是PlayCover，画质iPad Pro2024全高，轻松1080p120帧，连手柄玩的

**Comment by aino2333** [comment]
手机和电脑咯<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by ghost** [comment]
OPPOA5算什么！看我的（doge
![Screenshot_2024-09-24-12-51-32-50](https://github.com/user-attachments/assets/72b980fd-7ca1-42ff-99da-a29a2b193066)

**Comment by RWHwaxili** [comment]
Redmi note 12 turbo

**Comment by 1luik** [comment]
iqoo z8x

**Comment by grantchen668** [comment]
是我的伙伴「法厄同」给我提供的H.D.D.哦

**Comment by BJ-cmyk** [comment]
天选姬

**Comment by IHushuan** [comment]
拯救者

**Comment by KhlilFong** [comment]
尼康Z9
![Uploading 1730956641143.jpg…]()

**Comment by Sunset66** [comment]
当然是空洞深潜系统，也就是HDD。
